testfuzz/build.gradle

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+apply plugin: 'application'
+apply plugin: 'java-library'
+apply plugin: 'jacoco'
+
+jar {
+  archiveBaseName = 'besu-test-fuzz'
+  manifest {
+    attributes(
+      'Specification-Title': archiveBaseName,
+      'Specification-Version': project.version,
+      'Implementation-Title': archiveBaseName,
+      'Implementation-Version': calculateVersion()
+      )
+  }
+}
+
+dependencies {
+  implementation project(':besu')
+  implementation project(':crypto:algorithms')
+  implementation project(':datatypes')
+  implementation project(':ethereum:referencetests')
+  implementation project(':evm')
+  implementation project(':util')
+
+  implementation 'com.fasterxml.jackson.core:jackson-databind'
+  implementation 'com.gitlab.javafuzz:core'
+  implementation 'info.picocli:picocli'
+  implementation 'io.tmio:tuweni-bytes'
+  implementation 'org.jacoco:org.jacoco.agent'
+  implementation 'org.jacoco:org.jacoco.core'
+}
+
+application {
+  applicationName = 'BesuFuzz'
+  mainClass = 'org.hyperledger.besu.testfuzz.BesuFuzz'
+  applicationDefaultJvmArgs = [
+    '-javaagent:$APP_HOME/lib/jacocoagent.jar'
+  ]
+}
+
+tasks.register("fuzzAll", JavaExec) {
+  classpath = sourceSets.main.runtimeClasspath
+  mainClass = 'org.hyperledger.besu.testfuzz.BesuFuzz'
+  args = [
+    "eof-container",
+    "--tests-dir=${projectDir}/../ethereum/referencetests/src/reference-test/external-resources/EOFTests",
+    "--corpus-dir=${buildDir}/generated/corpus",
+    "--client=evm1=evmone-eofparse",
+    "--client=revm=revme bytecode",
+    // "--client=neth=netheofparse -x",
+    // "--client=geth=eofdump eofparser",
+    // "--client=etjs=tsx ../../../ethereumjs/ethereumjs-monorepo/packages/evm/scripts/eofContainerValidator.ts"
```

Comment:
are these comments left in for a reason

Reply:
I re-worked this section, so closing this comment as moot.